integration-tests/narayana-jta/src/test/java/io/quarkus/narayana/jta/TransactionalResourceIT.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.narayana.jta;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+class TransactionalResourceIT {
```

Comment:
The class name should not contain the "IT" which will be running with the failsafe. But we only want to run the native test with the integration test. So I suggest to rename this class to "TransactionalTestCase".

Also there is already a similar test in the https://github.com/quarkusio/quarkus/blob/master/integration-tests/main/src/test/java/io/quarkus/it/main/TransactionTestCase.java